mixer/test/client/env/envoy_conf.go

```diff
@@ -273,7 +273,7 @@ func (s *TestSetup) CreateEnvoyConf(path string, stress, faultInject bool, mfCon
 		AdminPort:       ports.AdminPort,
 		MixerServer:     fmt.Sprintf("localhost:%d", ports.MixerPort),
 		Backend:         fmt.Sprintf("localhost:%d", ports.BackendPort),
-		AccessLog:       "/dev/stdout",
+		AccessLog:       "/tmp/envoy-access.log",
```

Comment:
To avoid hard coding the file path for the envoy access log file, maybe you can add a new field (e.g. AccessLogPath) into [TestSetup](https://github.com/istio/istio/blob/master/mixer/test/client/env/setup.go#L31), and initialize this field in [NewTestSetup](https://github.com/istio/istio/blob/01674c30b4ad3b8e6255a6f5b76bc9cb22a1bf9c/mixer/test/client/env/setup.go#L71) to "/tmp/envoy-access.log". Then pass the value to config template at line 276 in this file to 
AccessLog: s.AccessLogPath.

Each test calls NewTestSetup ([Example](https://github.com/istio/istio/blob/0077c081b6e5cc0694b73a24195cfdca8eb0e6ed/pilot/pkg/proxy/envoy/v2/xds_test.go#L111)), If you want to specify another path in a test, then you can change it after the call to NewTestSetup(), but before SetUp().
testEnv.AccessLogPath = new-path.

@qiwzhang may have better ideas on this.